### ISSUE TITLE
Photo cropping fine tuning

### ISSUE THREAD

**Issue by TimMorganlive** [loogart/Project-Mountain#14]
While the photo cropping requirement has been addressed well, it would be important to find a way to better clarify the desired final product of the crop.  ie - some ppl crop just the face, some crop full shoulders.  It would be very beneficial to somehow guide them to crop our requirements consistently, rather than relying on them to read the instructions.
for example: perhaps the 'final crop' of the photo is blurred except for  an oval window in which to put their face.  (ref. www.idphoto4you.com)

**Comment by loogart** [comment]
@TimMorganlive check it out on the live prototype! let me know what you think.

https://loogart.github.io/Project-Mountain/registration-step2.html

**Comment by TimMorganlive** [comment]
This is looking amazing!  As in it is a 1000% improvement to the old
concept.  If I had to be honest, I would say that having the 'guide'
embedded into the cropping box would be easier to use, but like we
discussed, I think this mitigates a LOT, if not 99% of the cropping issues.

**Comment by loogart** [comment]
GOT IT! I managed to get the photo guide lines inside the actual cropper tool. Check it out. 

![screen shot 2017-10-31 at 11 07 28 am](https://user-images.githubusercontent.com/7881400/32231344-b615f386-be2b-11e7-98c1-7bc4abe91312.png)

**Comment by TimMorganlive** [comment]
we'll talk about actual sizing, but... HOLY $h!7!!!!  That looks
incredible!!!

**Comment by loogart** [comment]
@TimMorganlive yeah, just let me know what is the size/ratio/required spacing and I will try to fix it up, but I think for the most part, it works!

**Comment by TimMorganlive** [comment]
its a thing of beauty

**Comment by loogart** [comment]
add padding around head guides
